api-service/src/controllers/ConnectorInstanceCreate/ConnectorInstanceCreateValidationSchema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "enum": [
+                "api.connector-instance.create"
+            ]
+        },
+        "ver": {
+            "type": "string"
+        },
+        "ts": {
+            "type": "string"
+        },
+        "params": {
+            "type": "object",
+            "properties": {
+                "msgid": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "msgid"
+            ],
+            "additionalProperties": false
+        },
+        "request":{
+            "type":"object",
+            "properties": {
+                
+                "dataset_id":{
+                    "type":"string"
+
+                },
+                "connector_id":{
+                    "type":"string"
+                
+                }
```

Comment:
What about the other fields in the request, not just minimal we have to support all the fields with minimal fields as required

Reply:
i have updated the following required fields.
data_format and connector_config